packages/hash/api/src/graphql/resolvers/pages/updatePageContents.ts

```diff
@@ -111,41 +37,61 @@ export const updatePageContents: Resolver<
 
   return await dataSources.db.transaction(async (client) => {
     // Create any _new_ blocks
-    const newBlocks = await Promise.all(
-      actions
-        .map((action, i) => ({ action, i }))
-        .filter(({ action }) => action.insertNewBlock)
-        .map(({ action, i }) =>
-          createBlock(client, action.insertNewBlock!, user).catch((err) => {
-            if (err instanceof UserInputError) {
-              throw new UserInputError(`action ${i}: ${err}`);
-            }
-            throw err;
-          }),
-        ),
-    );
+    const newBlocks: Block[] = [];
+
+    /** @todo: find way to concurrently create blocks with outgoing links */
```

Comment:
Created [a task for this](https://app.asana.com/0/1201095311341924/1201817757879404/f)